.changeset/good-mails-sleep.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-kubernetes-backend': patch
+'@backstage/plugin-kubernetes-common': patch
+'@backstage/core-plugin-api': patch
+'@backstage/app-defaults': patch
+'@backstage/plugin-auth-backend': patch
+'@backstage/plugin-kubernetes': patch
```

Comment:
I think it's best to split this up into a couple of more changesets that describes the changes that were applied for each individual package.

Reply:
Sure thing I will work on this.